.github/workflows/flink-ci.yml

```diff
@@ -60,7 +60,37 @@ on:
     - 'CONTRIBUTING.md'
 
 jobs:
-  flink-tests:
+
+  # Only test the latest flink version with scala 2.11 for saving testing time.
+  flink-scala-2-11-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        jvm: [ 8, 11 ]
+        flink: [ '1.14' ]
+    env:
+      SPARK_LOCAL_IP: localhost
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.jvm }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - run: echo -e "$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)\t$(hostname -f) $(hostname -s)" | sudo tee -a /etc/hosts
+      - run: ./gradlew -DsparkVersions= -DhiveVersions= -DflinkVersions=${{ matrix.flink }} :iceberg-flink:iceberg-flink-${{ matrix.flink }}:check :iceberg-flink:iceberg-flink-runtime-${{ matrix.flink }}:check -DscalaVersion=2.11 -DknownScalaVersions=2.11 -Pquick=true -x javadoc
```

Comment:
This PR should merge before this PR : https://github.com/apache/iceberg/pull/4193 . Because if we don't merge the #4193 then this artifact name  `iceberg-flink-runtime-1.14` is not correct. (It  is `iceberg-flink-runtime-1.14_2.12`)